tools/release.js

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
+'use strict';
+
+const yargs = require('yargs/yargs');
+const { hideBin } = require('yargs/helpers');
+const readline = require('readline').createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
```

Comment:
`readline` is used to prompt the user for confirmation about pushing the tags later on